src/fitnesse/wikitext/parser/WikiSourcePage.java

```diff
@@ -127,4 +127,9 @@ private boolean isParentOf(WikiPage possibleParent) {
     public int compareTo(SourcePage other) {
         return getName().compareTo(other.getName());
     }
+
+    public WikiPage getPage() {
```

Comment:
This method creates a leak in the abstraction. From the parser point of view, it should not be aware of the `WikiPage` interface.